index.html

```diff
@@ -431,6 +431,7 @@ <h2 class="form-title">Contact me</h2>
               <textarea
```

Comment:
[OPTIONAL] You can consider adding descriptions to the labels of your form. You can hide the text in the labels with styles in your CSS stylesheet. This way they will be only accessible to screen readers only. [Read more about it here.](https://a11y-guidelines.orange.com/en/web/components-examples/accessible-hiding/)

Reply:
Hello so I've done the label and its hidden and only available for screen readers only using `sr-only`
Is there any other problem with my project?